packages/flutter/lib/src/cupertino/radio.dart

```diff
@@ -297,7 +293,7 @@ class _CupertinoRadioState<T> extends State<CupertinoRadio<T>> with TickerProvid
       checked: widget._selected,
       selected: accessibilitySelected,
       child: buildToggleable(
-        mouseCursor: effectiveMouseCursor,
+        mouseCursor: widget.mouseCursor ?? WidgetStateProperty.all(CupertinoRadio.defaultMouseCursor(widget.onChanged)),
```

Comment:
Instead of doing this here, can you set the default in the parameter list of the constructor?

Reply:
`defaultMouseCursor` takes `widget.onChanged` as an argument. Since `widget.onChanged` is non-`const`, `defaultMouseCursor` can't be set as the default `mouseCursor` in the parameter list.